Dockerfile

```diff
@@ -25,15 +25,27 @@ RUN if [ "$BUILDARCH" = "$TARGETARCH" ]; then \
         pip3 install mavsdk; \
     fi
 
-WORKDIR /fog-tools
-
 # make all commands in /fog-tools/* invocable without full path
 ENV PATH=$PATH:/fog-tools
 ENV PYTHONPATH=/usr/lib/python3.10/site-packages
 
 COPY src/ /fog-tools/
 
+# this dir is mounted to us by at least when running from fog-hyper's `$ fog tool` wrapper.
+# the point of it is to make it that if we write files (like recording a ROS bag or
+# downloading logs from Flight Controller), they end up in dir that is visible to debug web UI.
 WORKDIR /tools-data
 
-CMD ["ls", "-1", "/fog-tools"]
```

Comment:
This didn't work because it ran:

```
"bash" "-c" "ls" "-1" "/fog-tools"
```

which is very different to:

```
"bash" "-c" "ls -1 /fog-tools"
```